create_infotexts.py

```diff
@@ -144,68 +168,70 @@ def create_smvk_mm_link(item):
 
 
 def select_best_mapping_for_depicted_person(flipped_name):
-        """
+    """
```

Comment:
Incorrect indentation of docstring. Start with a descriptive line. Prefix `flipped_name` by `param`. Indent the `"surname...` bit once. Don't use double-spaces before "maps"